modules/integration/src/test/scala/scala/cli/integration/ConfigTests.scala

```diff
@@ -127,7 +127,7 @@ class ConfigTests extends ScalaCliSuite {
           TestUtil.putCsInPathViaEnv(root / "bin")
 
       val res = os.proc(TestUtil.cli, "--power", "config", "httpProxy.address")
-        .call(cwd = root, env = extraEnv)
+        .call(cwd = root, env = extraEnv, mergeErrIntoOut = false)
```

Comment:
?

Reply:
wow, that was an experiment I did to confirm what the default behavior was, then forgot to remove it.